Makefile

```diff
@@ -140,13 +140,6 @@ $(CODE_GEN_DIR):
 
 ##@ Development
 
-## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects,
-## then Kustomize the CustomResourceDefinition objects for the 'crd' package to use.
-.PHONY: manifests
-manifests: controller-gen kustomize
-	$(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="./api/..." output:crd:artifacts:config=config/crd/bases
```

Comment:
We still need a `make` target that generates the CRDs, right?

Reply:
Not for the core-helm chart.
What do you think is using that YAML?

Reply:
KubeStellar certainly needs to maintain its CRDs! If I recall correctly, the KubeStellar controller-manager establishes some in each WDS and the OCM Status Add-On Controller establishes some in the ITS.